lib/matplotlib/collections.py

```diff
@@ -2042,10 +2043,26 @@ def draw(self, renderer):
             renderer.draw_gouraud_triangles(
                 gc, triangles, colors, transform.frozen())
         else:
-            renderer.draw_quad_mesh(
-                gc, transform.frozen(), self._meshWidth, self._meshHeight,
-                coordinates, offsets, transOffset, self.get_facecolor(),
-                self._antialiased, self.get_edgecolors())
+            try:
+                sig = inspect.signature(renderer.draw_quad_mesh)
```

Comment:
As noted above I really don't think this is the correct (extensible) approach.  Hence blocking, but just as a marker of "this specific point needs more discussion".